package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "tawk-magento-2",
```

Comment:
Add `private : true` to avoid accidental publish to npm

Reply:
Done!